templates/aem-multimodule-project/content/src/main/content/META-INF/vault/filter.xml

```diff
@@ -4,6 +4,9 @@
 <% if (includeAcsAemCommons && enableErrorHandler) { %>
     <filter root="/apps/sling/servlet/errorhandler"/>
 <% } %>
+<% if (createEditableTemplatesStructure) { %>
+    <filter root="/conf/${appsFolderName}"/>
```

Comment:
IIUC, this is incorrect. With this in place, installation of the package will overwrite any existing templates, which really makes them non-editable (sure, you can edit them, but the next deployment will blow away your edits). TBH, I'm not 100% sure what the right answer is to this.

One choice would be to exclude the "editable" directories from the package. Another would be to use merge.